### DISCUSSION TITLE
Make Environment settings available for Organizations

### DISCUSSION THREAD

**Discussion by GoranHalvarsson** [community/community#20940] [Enterprise]
Please make Environment settings available for Organizations :pensive:

**Comment by thaisonzdx** [comment]
It's available for public repository only. If you wanna use it on private repos, you must upgrade your organization to Enterprise plan, to get more informations, check this [link](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment)

> Note: To create an environment in a private repository, your organization must use GitHub Enterprise Cloud. For more information about how you can try GitHub Enterprise Cloud for free, see "[Setting up a trial of GitHub Enterprise Cloud](https://docs.github.com/en/get-started/signing-up-for-github/setting-up-a-trial-of-github-enterprise-cloud)."

**Reply by jmbenedetto** [reply]
Hi! I have only public repos, but do not see the "Environment" option at the organization level. Would you know it it should be available?
